lib/cocoapods-core/specification/dsl.rb

```diff
@@ -677,8 +677,19 @@ def dependency(*args)
           end
         end
         unless version_requirements.all? { |req| req.is_a?(String) }
-          raise Informative, 'Unsupported version requirements'
+          version_requirements.each do |requirement|
+            if requirement.is_a?(Hash)
+              if !requirement[:path].nil?
+                raise Informative, 'Podspecs can only use remote pods as dependencies. :path is not supported'
+              elsif !requirement[:git].nil?
+                raise Informative, 'Podspecs can only use remote pods as dependencies. :git is not supported'
```

Comment:
The error message isn't _technically_ correct - you can use `:git` dependencies, but they cannot be specified by the podspec itself (ex. if the pod file contains `pod 'MyPod', :git => '...'`, a podspec with the dependency `'MyPod'` will be fulfilled by that installation)

What about the following:

> Podspecs cannot specify the source of dependencies. The `:git` option is not supported

Reply:
After rereading this for a couple of times, I think In understand what you mean.

Will change the message.

Reply:
Is it the same with path?

Reply:
yup `:path` behaves the same as far as I'm aware